sa/storage-authority.go

```diff
@@ -229,6 +230,73 @@ func (ssa *SQLStorageAuthority) GetLatestValidAuthorization(registrationID int64
 	return ssa.GetAuthorization(auth.ID)
 }
 
+// incrementIP increments the byte within `ip` at `index`, accounting for overflow.
+// Index must be greater than zero and less than len(ip).
+func incrementIP(ip *net.IP, index int) {
+	if (*ip)[index] == 255 {
+		(*ip)[index-1]++
```

Comment:
What happens if `(*ip)[index-1]` is 255 before this?

Boy, it sure feels like there'd be a helper for capturing a range of IPs already somewhere.

Alternatively, perhaps converting the []byte into an integer and then doing the math on that and converting back to []byte require less work?

Oh, actually, now that I look at this, maybe we just convert all ipv4 addresses to their ipv6 equivalent and just do that work?